### PR TITLE
Update cl.m3u

### PR DESCRIPTION
**Pull request by sguinetti**
This PR is for add new streams from Chile.

### CODE REVIEW

**Review by its0x08 — DISMISSED** [review]

**Review by freearhey — DISMISSED** [review]

File: streams/cl.m3u [hunk 1]
```diff
@@ -97,6 +97,8 @@ https://stmv.panel.mivideo.pro/ekiztv/ekiztv/playlist.m3u8
 https://paneltv.online:1936/8154/8154/playlist.m3u8
 #EXTINF:-1 tvg-id="ElPinguinoTV.cl" status="online",El Pingüino TV (720p)
 https://github.com/MachineSystems/archived_m3u8/raw/main/elpinguino_cl.m3u8
+#EXTINF:-1 tvg-id="EnerGeek3.cl" status="online",EnerGeek Señal 2 (720p) [Not 24/7]
```
Comment (LaneSh4d0w) [inline streams/cl.m3u:100]: Signal 2 or 3 ?

File: streams/cl.m3u [hunk 2]
```diff
@@ -385,11 +391,37 @@ http://159.69.56.148:25461/live/Mix24-7Gratis/78qy9/6.m3u8
...
 http://159.69.56.148:25461/live/Mix24-7Gratis/78qy9/25.m3u8
-#EXTINF:-1 tvg-id="" status="error",Camara de Diputados (DVR) (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="CDTV.cl" status="error",CDTV (DVR) (720p) [Not 24/7]
 http://camara.02.cl.cdnz.cl/cdndvr/ngrp:live_all/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="error",CHV Noticias (720p) [Not 24/7]
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCRsUoZYC1ULUspipMRnMhwg/live
 #EXTINF:-1 tvg-id="" status="error",Estación TV (Chillán) (720p) [Not 24/7]
 https://pantera1-100gb-cl-movistar.dps.live/estaciontv/estaciontv.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="error",Spectrum Channel LGBTQ+ (480p) [Not 24/7]
+#EXTINF:-1 tvg-id="SpectrumChannel.cl" status="error",Spectrum Channel (480p) [Not 24/7]
```
Comment (LaneSh4d0w) [inline streams/cl.m3u:400]: Now named SMAS, under the spectrumchannel.cl website.

File: streams/cl.m3u [hunk 3]
```diff
@@ -385,11 +391,37 @@ http://159.69.56.148:25461/live/Mix24-7Gratis/78qy9/6.m3u8
...
 https://vdohd.cl:1936/8078/8078/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/cl.m3u:401]:
```suggestion
https://panel.miplay.cl:8081/spectrumchannel/index.m3u8
```

File: streams/cl.m3u (hunk 1 above)
Comment (sguinetti) [reply streams/cl.m3u]: This is 3.

File: streams/cl.m3u (hunk 2 above)
Comment (sguinetti) [reply streams/cl.m3u:400]: This seems similar with Energeek and Kanade. They come together but then split up and go their separate ways.

**Review by BellezaEmporium — APPROVED** [review]

**Review by boten99 — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**